documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,150 @@ You can add {a-rhv} source provider by using the {ocp} web console.
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Adding a Red Hat {virt} source provider
+
+You can add a Red Hat {virt} source provider by using the {ocp} web console.
+
+.Prerequisites
+
+* TBD
+endif::[]
 
 .Procedure
 
 . In the {ocp} web console, click *Migration* -> *Providers for virtualization*.
 . Click *Create Provider*.
 ifdef::vmware[]
-. Select *VMware* from the *Provider type* list.
+. Click *VMware vSphere*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *vCenter host name or IP address*: vCenter host name or IP address - if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *vCenter user name*: vCenter user, for example, `user@vsphere.local`
-* *vCenter password*: vCenter user password
-* *VDDK init image*: VDDKInitImage path
+* *Provider resource name*: Name to display in the list of providers.
+* *URL*: Endpoint.
+* *VDDK init image*: VDDKInitImage path.If left empty, some functionality will not be available.
+* *Username*: vCenter user, for example, `user@vsphere.local`.
+* *Password*: vCenter user password.
+* *SHA-1 fingerprint*: [TBD].
+
+. Choose one of the following options for validating a CA certificate:
 
-. To allow a migration without validating the provider's CA certificate, select the *Skip certificate validation* check box. By default, the checkbox is cleared, meaning that the certificate will be validated.
-. Enter the *SHA-1 fingerprint*.
+** *Skip validation* : Migrate without validating a CA certificate.
+** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
 endif::[]
 ifdef::rhv[]
-. Select *Red Hat Virtualization* from the *Provider type* list.
+. Click *Red Hat Virtualization*
+. Specify the following fields:
+thats
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: Endpoint
```

Comment:
Endpoint -> API endpoint of RHVM

Reply:
@yaacov -- for UI, also.

Reply:
@ahadas Suggested rephrasing: "API endpoint of the virtual machine being imported from {rhv-short}"